common/include/pcl/conversions.h

```diff
@@ -67,12 +63,12 @@ namespace pcl
 
       template<typename U> void operator() ()
       {
-        pcl::PCLPointField f;
-        f.name = traits::name<PointT, U>::value;
-        f.offset = traits::offset<PointT, U>::value;
-        f.datatype = traits::datatype<PointT, U>::value;
-        f.count = traits::datatype<PointT, U>::size;
-        fields_.push_back (f);
+        fields_.emplace_back(PCLPointField{
```

Comment:
Why are you invoking the copy ctor instead of the ctor?

Reply:
Play around here https://godbolt.org/z/6u-sAT
In a nutshell
```cpp
v.emplace_back("", 1, 2, 3); // doesn't compile/link. the ctor does not exist. only in C++20
v.emplace_back({"", 1, 2, 3}); // doesn't compile. You can't use aggregate initalization inside emplace back. Syntax works with push_back
```
The second case, regardless of the option I end up using will still trigger aggregate initalization  followed by a move ctor. I will likely turn it into a push_back to convey the exact idea of what is happening.

Reply:
> ctor does not exist. only in C++20

That's a bit sad.

I also think `push_back` will be better